src/main/kotlin/lotto/view/OutputView.kt

```diff
@@ -0,0 +1,40 @@
+package lotto.view
+
+import lotto.domain.Lottos
+import lotto.domain.Prize
+import java.text.DecimalFormat
+
+class OutputView {
+
+    fun printGeneratedLottos(lottos: Lottos) {
+        println("${lottos.items.size}개를 구매했습니다.")
+        lottos.items.forEach { it ->
+            val sorted = it.lottoNumbers.map { it.number }.sorted()
+            println("[${sorted.joinToString(", ")}]")
+        }
```

Comment:
```suggestion
        lottos.items.forEach { lotto ->
            val sorted = lotto.lottoNumbers.map { it.number }.sorted()
            println("[${sorted.joinToString(", ")}]")
        }
```

It might be less readable since there are two `it`s.
What about giving it a name if you're not going to implicit it anyways?

Reply:
```suggestion
        lottos.items.forEach { lotto ->
            lotto.lottoNumbers
                .map { it.number }
                .sorted()
                .also { println(it) }
        }
```

if you use a [scope function](https://kotlinlang.org/docs/scope-functions.html) might be more readable

Reply:
```suggestion
        lottos.items.forEach {
            println(it.lottoNumbers.map(LottoNumber::number).sorted())
        }
```

or maybe this?

Reply:
What a kind takiyaki ...

Reply:
Wow the last one made me shocked

Reply:
### 📝 How List.toString() is actually implemented (in Kotlin/Java) !!

In Kotlin, a List usually implements the kotlin.collections.List interface, which is backed by Java’s java.util.List or AbstractList.

The actual toString() implementation comes from Java’s AbstractCollection, which many Kotlin collections inherit from (especially for JVM targets).

Here's a simplified version of how AbstractCollection.toString() is implemented:

```kotlin
public String toString() {
    Iterator<E> it = iterator();
    if (!it.hasNext())
        return "[]";

    StringBuilder sb = new StringBuilder();
    sb.append('[');
    for (;;) {
        E e = it.next();
        sb.append(e == this ? "(this Collection)" : e);
        if (!it.hasNext())
            return sb.append(']').toString();
        sb.append(',').append(' ');
    }
}
```

So:

- If the list is empty → returns "[]"
- Otherwise → appends each element’s toString() result, separated by commas, and wraps them in brackets